dbgl.spec

```diff
@@ -123,6 +123,13 @@ appstream-util validate-relax --nonet \
 %{_datadir}/appdata/%{name}.appdata.xml
 
 %changelog
+
+* Sun Nov 05 2017 Unitedrpms Project <unitedrpms AT protonmail DOT com> 0.82-1  
```

Comment:
Again this s**t repo? Replace this row please by your own.

Reply:
agree with @xvitaly , please use some real person as email.